### ISSUE TITLE
Update CKEditor button images

### ISSUE THREAD

**Issue by herbdool** [backdrop/backdrop-issues#4026]
**Description of the bug**
The CKeditor buttons in the admin settings look different from the front end. I first noticed it with the Table button and the Source button.

The admin page:
![Screenshot from 2019-09-10 22-35-04](https://user-images.githubusercontent.com/1071483/64663919-5febb880-d41b-11e9-9cb2-1c30926394f4.png)

Add a node form:
![Screenshot from 2019-09-10 22-34-51](https://user-images.githubusercontent.com/1071483/64663918-5febb880-d41b-11e9-89d9-b3ae43cf04eb.png)

I viewed this in 1.14.x and 1.13.x.

**Comment by BWPanda** [comment]
Hmm, interesting! Confirmed in latest 1.18.x...

**Comment by indigoxela** [comment]
It seems like the "admin buttons" in core/modules/ckeditor/images/buttons/ should be updated to the recent versions from core/misc/ckeditor/plugins. :wink:

**Comment by BWPanda** [comment]
I started doing this [PR](https://github.com/backdrop/backdrop/pull/3378), only to get to the end of the images and realise that the source I was using didn't have all the ones I needed... So adding my PR here so as not to lose all my work, but this still needs work.

**Comment by indigoxela** [comment]
> only to get to the end of the images and realise that the source I was using didn't have all the ones I needed

I'm not sure, but it might be that CKEditor is using [sprite images](https://github.com/backdrop/backdrop/blob/1.x/core/misc/ckeditor/plugins/icons_hidpi.png).

**Comment by BWPanda** [comment]
Yes, that's what I used for the PR changes, but there aren't as many images there as are in https://github.com/backdrop/backdrop/tree/1.x/core/modules/ckeditor/images/buttons...

**Comment by indigoxela** [comment]
> but there aren't as many images there as are in core/modules/ckeditor/images/buttons...

I see. Maybe they're not in use anymore? But it's probably easiest if you replace the ones you have available in the sprite and keep the other (older) ones.
